Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Writers/TokenType.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Amazon.Lambda.Annotations.SourceGenerator.Writers
+{
+    /// <summary>
+    /// These enums dictate the deserialized <see cref="YamlNode"/> type during the invocation of YamlWriter.SetToken(..)
+    /// These token types do not play any roles while using jsonWriter.SetToken(..)
+    /// </summary>
+    public enum TokenType
+    {
+        /// <summary>
+        /// This token type is deserialized to YamlSequenceNode
+        /// </summary>
+        List,
+
+        /// <summary>
+        /// This token type is deserialized to YamlMappingNode
+        /// </summary>
+        KeyVal,
+
+        /// <summary>
+        /// This token type is deserialized to YamlMappingNode
+        /// </summary>
+        Object,
+
+        /// <summary>
+        /// This token type is deserialized to YamlScalarNode
+        /// </summary>
+        Other
```

Comment:
documentation for these values is confusing. The same line is repeated for KeyVal and Object. Also, what is YamlMappingNode

Reply:
They are intended to be the same for KeyVal and Object. I have added a `<see cref ="" />` for these YAML entities.